client/containers/AddNewPostContainer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import AddPostForm from '../components/AddPostForm'
+import { addNewPost } from '../actions'
+
+const mapStateToProps = state => {
+  return {}
```

Comment:
If we're not mapping any states to props, we can remove this function and just pass `null` to the `connect` function below.